docs/api/apiv3/components/schemas/project_phase_definition_collection_model.yml

```diff
@@ -0,0 +1,47 @@
+# Schema: ProjectPhaseDefinitionCollectionModel
+---
+allOf:
+  - $ref: './collection_model.yml'
+  - type: object
+    required:
+      - _links
+      - _embedded
+    properties:
+      _links:
+        type: object
+        required:
+          - self
+        properties:
+          self:
+            allOf:
```

Comment:
The `allOf` is not correct here. In the self link, only `href` and `title` will be returned. But it seems to be a common mistake throughout the specification @Kharonus. E.g. the ProjectsCollection documentation states the same. Accepting duplication, it might be better to provide another type of link that is scoped down so that documenting people can more easily reference that.